control-station/package.json

```diff
@@ -7,6 +7,8 @@
 		"dev": "vite",
 		"build": "tsc && vite build",
 		"lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
Thanks for working on this! Quick question - Is there a specific reason for setting the 'max-warnings' option to 0 in the 'lint' script?

Reply:
This lint script came from the Vite template during project initialization, but to answer, setting the threshold ensures any warnings cause the `eslint` program to error as opposed to announcing the error but still finishing successfully (exit code 0). We want the lint check to fail on warnings accordingly. Refer to [`--max-warnings` in the ESLint CLI Reference](https://eslint.org/docs/latest/use/command-line-interface#--max-warnings) for more detail.